apps/src/StudioApp.js

```diff
@@ -1433,14 +1435,21 @@ StudioApp.prototype.displayFeedback = function (options) {
       this.maxRecommendedBlocksToFlag_);
   } else {
     // update the block hints lightbulb
-    const missingBlockHints = this.feedback_.getMissingBlockHints(this.requiredBlocks_.concat(this.recommendedBlocks_), options.level.isK1);
+    const missingBlockHints = this.feedback_.getMissingBlockHints(
+      this.requiredBlocks_.concat(this.recommendedBlocks_),
+      options.level.isK1,
+    );
     this.displayMissingBlockHints(missingBlockHints);
 
     // communicate the feedback message to the top instructions via
     // redux
     const message = this.feedback_.getFeedbackMessage(options);
     getStore().dispatch(setFeedback({ message }));
   }
+
+  if (experiments.isEnabled('onetimeHintPrompt')) {
```

Comment:
@ryansloan do we want this behind an experiments flag initially?

Reply:
Nope! :shipit: